SECURITY.md

```diff
@@ -5,7 +5,11 @@ The Corona-Warn-App is built with security and data privacy in mind to ensure yo
 **Please do not report security vulnerabilities directly on GitHub. GitHub Issues can be publicly seen and therefore would result in a direct disclosure.**
 
 * Please address questions about data privacy, security concepts, and other media requests to the corona-warn-app.opensource@sap.com mailbox.
-* For reporting a vulnerability, please use the Vulnerability Report Form for Security Researchers on [SAP Trust Center](https://www.sap.com/about/trust-center/incident-management.html). Please select "Corona-Warn-App" in the _product_ list. In the _versions_ field, either note the specific [release version](https://github.com/corona-warn-app/cwa-server/releases) or commit id of the master branch you investigated. The affected repository should be mentioned in the vulnerability description.
+* For reporting a vulnerability, please use the Vulnerability Report Form for Security Researchers on [SAP Trust Center](https://www.sap.com/about/trust-center/incident-management.html). 
```

Comment:
should we link https://vulnerability-form.cfapps.sap.hana.ondemand.com/#/ directly? Saves a click.

Reply:
The Trust Center also provides further information about security issue reporting, handling, disclosure guidelines, etc., which could be of interest to security researchers. Also, the Trust Center URL is less likely to change than the one for the concrete submission form. The colleagues would therefore appreciate the link to this page and not the form directly.